selfdrive/ui/translations/main_ar.ts

```diff
@@ -876,310 +876,310 @@ This may take up to a minute.</source>
     </message>
     <message>
         <source>DOWNLOAD</source>
-        <translation>تحميل</translation>
+        <translation>تنزيل</translation>
     </message>
     <message>
         <source>update available</source>
         <translation>يتوفر تحديث</translation>
     </message>
     <message>
         <source>never</source>
-        <translation>أبداً</translation>
+        <translation>إطلاقاً</translation>
     </message>
     <message>
         <source>up to date, last checked %1</source>
-        <translation>محدث، آخر فحص %1</translation>
+        <translation>أحدث نسخة، آخر تحقق %1</translation>
     </message>
 </context>
 <context>
     <name>SshControl</name>
     <message>
         <source>SSH Keys</source>
-        <translation>SSH Keys</translation>
+        <translation>مفاتيح SSH</translation>
     </message>
     <message>
         <source>Warning: This grants SSH access to all public keys in your GitHub settings. Never enter a GitHub username other than your own. A comma employee will NEVER ask you to add their GitHub username.</source>
-        <translation>تحذير: هذا يمنح SSH الوصول إلى جميع المفاتيح العامة في إعدادات GitHub. لا تدخل أبدًا اسم مستخدم GitHub بخلاف اسم المستخدم الخاص بك. لن يطلب منك موظف comma أبدًا إضافة اسم مستخدم GitHub الخاص به.</translation>
+        <translation>تنبيه: هذا يمنح SSH إمكانية الوصول إلى جميع المفاتيح العامة في إعدادات GitHub. لا تقم بإدخال اسم مستخدم GitHub بدلاً من اسمك. لن تطلب منك comma employee إطلاقاً أن تضيف اسم مستخدم GitHub الخاص بهم.</translation>
     </message>
     <message>
         <source>ADD</source>
-        <translation>أضف</translation>
+        <translation>إضافة</translation>
     </message>
     <message>
         <source>Enter your GitHub username</source>
-        <translation>أدخل اسم مستخدم GitHub الخاص بك</translation>
+        <translation>ادخل اسم المستخدم GitHub الخاص بك</translation>
     </message>
     <message>
         <source>LOADING</source>
-        <translation>جار التحميل</translation>
+        <translation>يتم التحميل</translation>
     </message>
     <message>
         <source>REMOVE</source>
-        <translation>نزع</translation>
+        <translation>إزالة</translation>
     </message>
     <message>
         <source>Username &apos;%1&apos; has no keys on GitHub</source>
-        <translation>لا يحتوي اسم المستخدم &apos;%1&apos; على مفاتيح على GitHub</translation>
+        <translation>لا يحتوي اسم المستخدم &apos;%1&apos; أي مفاتيح على GitHub</translation>
     </message>
     <message>
         <source>Request timed out</source>
-        <translation>انتهت مهلة الطلب</translation>
+        <translation>انتهى وقت الطلب</translation>
     </message>
     <message>
         <source>Username &apos;%1&apos; doesn&apos;t exist on GitHub</source>
-        <translation>اسم المستخدم &apos;%1&apos; غير موجود على GitHub</translation>
+        <translation>اسم المستخدم @apos;%1&apos; غير موجود على GitHub</translation>
```

Comment:
```suggestion
        <translation>اسم المستخدم &apos;%1&apos; غير موجود على GitHub</translation>
```

typo? if so, can you write a quick test to catch this in [test_translations.py](https://github.com/commaai/openpilot/blob/master/selfdrive/ui/tests/test_translations.py)?

Reply:
```
======================================================================
FAIL: test_entities_error (__main__.TestTranslations.test_entities_error) (name='العربية', file='main_ar')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/blah/github/openpilot/./selfdrive/ui/tests/test_translations.py", line 122, in test_entities_error
    self.assertEqual(len(matches), 0, f"The string(s) {matches} were found with '@' instead of '&'")
AssertionError: 1 != 0 : The string(s) ['apos'] were found with '@' instead of '&'

----------------------------------------------------------------------
Ran 7 tests in 0.558s
```